Current_Bylaws.md

```diff
@@ -92,4 +92,51 @@ Last Amended: March 5, 2021
+This organization hereby wishes to seek ASUC sponsorship. This organization adopts and incorporates by reference Schedule A of ASUC Bylaw 2201, and the same as may be amended from time by time, completely and fully as part of this constitution and its bylaws. This organization cannot amend this clause unless and until the ASUC has certified that this organization has ceased to be ASUC-sponsored, all ASUC property has been returned to the ASUC, access to all ASUC resources and facilities has been relinquished, and any debts, liabilities, or other responsibilities to the ASUC have been resolved. This organization will continue to be bound by any provisions specified in Schedule A for actions it took while this clause was in effect, even if this clause is amended.
+
+### Schedule A 
+
+1 This organization is organized and operated exclusively for nonprofit purposes. The property of this organization is irrevocably dedicated to the purposes specified in its primary governing document and no part of the net income or assets of this organization shall ever inure to the benefit of any director, officer, or member thereof or to the benefit of any private person. 
+
+2 This organization recognizes that current or past ASUC sponsorship or funding does not imply a guarantee of continued or future sponsorship or funding and that status as a university registered student organization does not guarantee status as an ASUC-sponsored student organization or ASUC funding.
+
+3 The ultimate constitutional, budgetary, and policy making authority of this organization shall be vested entirely and exclusively in ASUC members, at least a majority of whom shall be undergraduate students. Only ASUC members may hold officer positions in this organization. This section shall not be interpreted to prevent non-ASUC members from holding advisory positions in the organization.
+
+4 This organization is not a unit of the ASUC or of the University of California. Unless otherwise specified in its primary governing document, this organization shall be considered an unincorporated association organized under the laws of California. 
+
+5 All funds, facilities, and resources allocated or provided to this organization by the ASUC (including the Graduate Assembly) during the term of its sponsorship shall remain the property of the ASUC at all times. If the organization ceases to be sponsored or ceases to exist, all ASUC property and unspent funds shall be immediately returned to the ASUC (including the Graduate Assembly).
+
+5.1 Any property [## other long term property / subscriptions] purchased by this organization using funds provided to it by the ASUC (including the Graduate Assembly) shall be the property of the ASUC. The ASUC shall retain ownership proportional to its contribution in any property purchased partially with ASUC funds.
```

Comment:
what are the brackets for?

Reply:
its copy pasted from asuc so its ok to leave them